scss/movieDetails.scss

```diff
@@ -94,6 +94,32 @@
 		margin-top: 3rem;
 	}
 
+	.movie-rating-hats {
+		list-style-type: none;
+		display: flex;
+		padding-bottom: 0.2rem;
+		justify-content: center;
+		
```

Comment:
Yeah, totally forgot to test it on other browsers then Firefox and on mobile view, since we are not focusing on the css...
Fixed it now, hopefully :fearful:

Reply:
Looking good! 👌